### ISSUE TITLE
Sort by passage titles

### ISSUE THREAD

**Issue by kpalik** [mcdemarco/prepub#2]
Is it possible to add an option "Sort by Pasage Titles" on options page? I was using some prefixes in names my passages, like:

E1S1P1 (Episode 1, Scene 1, Passage 1)
E1S2P1 (Episode 1, Scene 2, Passage 1)
E1S2P2 (Episode 1, Scene 2, Passage 2)

and so on

now if i add Passage 2 to Episode 1 Scene 1 it will land on end of the exported file. It is hard to proofread that kind of file if passages are "randomly" placed in downloaded document.

**Comment by mcdemarco** [comment]
Are you getting the random ordering from writing in the Twine UI?  (Just curious.)  

I'll add this soon.

**Comment by kpalik** [comment]
Yes, I worked in Twine UI, exported to markdown, then used prepub to generate PDF (it preserves the internal links, which was crucial for us), and then worked on Adobe Acrobat for Google Drive plugin to proofread and comment the file, that was our workflow for about 20+ iterations on 200+ pages document

for now, this feature is a low priority for us because our team has moved to the next phase of the project, and scenario proofreading is more or less closed
